data/macos.yaml

```diff
@@ -532,6 +532,20 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Recent_Items'
 ---
+name: MacOSSafariDownloads
```

Comment:
Please move this definition to webbrowser.yaml and rename it to SafariDownloads

add it also to BrowserHistory

what about safari on Windows, any idea what the path of this file is?